tests/integration/SourceAdapters/SourceAdapterTest.php

```diff
@@ -95,8 +117,12 @@ public function test_it_adapts_file_contents($adapter, $source)
     {
         $adapter = new $adapter($source);
         $contents = $adapter->contents();
-        $this->assertTrue(is_resource($contents));
-        $this->assertEquals(get_resource_type($contents), 'stream');
+
+        if (is_resource($contents)) {
```

Comment:
Also this conditional seems like a code smell, because of the contents returned by the `StreamInterface` or the resource.